.github/workflows/optional.yml

```diff
@@ -30,12 +32,14 @@ jobs:
             "
             apt-get -y update &&
             DEBIAN_FRONTEND=noninteractive apt-get -y install build-essential cmake curl gettext ninja-build locales-all cpanminus git attr libattr1-dev  &&
-            git clone --depth=1 https://github.com/neovim/neovim.git &&
+            useradd --create-home qemuci &&
```

Comment:
Hmm, is this the same problem as with freebsd aka some tests require non-sudo or regular user access to work properly? Could be worth dropping a small comment on why it's needed IMO, easy to forget these things.